conf/iridanext.config

```diff
@@ -4,13 +4,13 @@ iridanext {
         path = "${params.outdir}/iridanext.output.json.gz"
         overwrite = true
         files {
-            samples = ["${params.outdir}/input/*_error_report.csv"]
+            samples = ["**/input/*_error_report.csv"]
         }
         metadata {
             idkey = "id_irida"
             samples {
                 csv {
-                    path = "${params.outdir}/filter/new_addresses.csv"
+                    path = "**/filter/new_addresses.csv"
                     idcol = "id"
```

Comment:
This doesn't work currently, because the file "new_addresses.csv" outputs the `meta.id` and we need the `meta.irida_id` to have it work with IRIDA-Next. I will do something like what was done in [staramrnf PR](https://github.com/phac-nml/staramrnf/pull/28/commits/5544cf5d02c447be456919607256859b8d320ca5) where another column is added to the output file to allow for scraping of information from the csv.

Reply:
I was able to make the fix to allow metadata be taken from `new_addresses.tsv` (switched to tsv due to ease of use with csvtk) so now the file outputs with an extra column with the `sample` and `sample_name`.  a246014